src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentCost = 40;
+  const costPerDays = days * rentCost;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days < 3) {
+    return costPerDays;
+  }
+
+  if (days < 7) {
```

Comment:
The number of days could also change, let's move them to the variables

Reply:
**Fixed**